llvmlite/tests/test_binding.py

```diff
@@ -3075,6 +3141,36 @@ def test_run(self):
         self.assertIn("%.4", orig_asm)
         self.assertNotIn("%.4", optimized_asm)
 
+    def test_run_O0(self):
```

Comment:
I think all test names are snake case with no capitals:
```suggestion
    def test_run_o0(self):
```

Also I'd be inclined to factor out the run functionality into a separate function like:

```python
    def run_o_n(self, level):
        pb = self.pb(level)
        mod = self.module()
        fun = mod.get_function("sum")
        orig_asm = str(fun)
        fpm = pb.getFunctionPassManager()
        fpm.run(fun, pb)
        optimized_asm = str(fun)
        return orig_asm, optimized_asm
```

then use it as in the following comment's suggestion...